go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/operator-framework/operator-lifecycle-manager
 
-go 1.21
+go 1.22
```

Comment:
I was expecting to see toolchain directive here too.

Reply:
great call out! I didn't know about toolchains. Looking at the [documentation](https://go.dev/doc/toolchain#get), I think it says that if you don't specify a toolchain it will take tool chain for that version:

```
If the toolchain line is omitted, the module or workspace is considered to have an implicit toolchain goV line, where V is the Go version from the go line.

For example, a go.mod that says go 1.21.0 with no toolchain line is interpreted as if it had a toolchain go1.21.0 line.
```

Should we start updating the Go z-version and as well, do you think?

Reply:
Good to know about implicit toolchain. I asked because I think `go mod tidy` adds it by default.

I think explicit is better than implicit, but I think we can go ahead with this PR.

Reply:
I agree. I'll make it explicit.